soh/soh/Enhancements/tts/tts.cpp

```diff
@@ -1148,6 +1146,47 @@ void RegisterOnSetGameLanguageHook() {
     GameInteractor::Instance->RegisterGameHook<GameInteractor::OnSetGameLanguage>([]() { InitTTSBank(); });
 }
 
+void RegisterOnSetDoAction() {
+    GameInteractor::Instance->RegisterGameHook<GameInteractor::OnSetDoAction>([](uint16_t action) {
+        if (CVarGetInteger(CVAR_SETTING("A11yTTS"), 0)) {
+            uint8_t language = CVarGetInteger(CVAR_SETTING("Languages"), 0);
+            const char* text;
+            switch (action) {
+                case DO_ACTION_CHECK:
+                    text = language == LANGUAGE_FRA ? "voir" : language == LANGUAGE_GER ? "lesen" : "check";
+                    break;
+                case DO_ACTION_ENTER:
+                    text = language == LANGUAGE_FRA ? "enter" : language == LANGUAGE_GER ? "kriechen" : "ranger";
+                    break;
+                case DO_ACTION_OPEN:
+                    text = language == LANGUAGE_FRA ? "ouvrir" : language == LANGUAGE_GER ? "öffnen" : "open";
+                    break;
+                case DO_ACTION_THROW:
+                    text = language == LANGUAGE_FRA ? "jeter" : language == LANGUAGE_GER ? "werfen" : "throw";
+                    break;
+                case DO_ACTION_CLIMB:
+                    text = language == LANGUAGE_FRA ? "monter" : language == LANGUAGE_GER ? "hinauf" : "climb";
+                    break;
+                case DO_ACTION_DROP:
```

Comment:
this seems to only come up when holding a bombchu (code seems to have it also apply with cuccos & master sword pedestal somehow, but I couldn't get drop action to show up with them)

should maybe be removed, along with throw